apps/customer/urls.py

```diff
@@ -12,5 +12,6 @@
 
 urlpatterns = [
     path('', include(router.urls)),
-    
+    path('customer-order/', TestCustomerCreateViews.as_view(), name='customer-order-create'),
+    path('customer-order/<str:pk>/', TestCustomerCreateViews.as_view(), name='customer-order-details'),    
```

Comment:
Why it is customer-order it should be customers/

Reply:
updated